test

```diff
@@ -79,27 +81,30 @@ idxr server config 2>&1 |
 # Indexer server cli subcommand works
 idxr server cli --help > /dev/null
 
+# Temporary hack - create startup-blocks direct if it doesn't exist
+mkdir $HOME/.mina-indexer/startup-blocks || true
+
 # Indexer server config passes cli with minimal args
 idxr server cli \
-    --initial-ledger tests/ledgers/mainnet-genesis.json \
-    --is-genesis-ledger \
-    --root-hash 3NKeMoncuHab5ScarV5ViyF16cJPT4taWNSaTLS64Dp67wuXigPZ &
-IDXR_PID=IDXR_PID=$(echo $!)
-kill $IDXR_PID
+     --initial-ledger tests/ledgers/mainnet-genesis.json \
+     --is-genesis-ledger \
+     --root-hash 3NKeMoncuHab5ScarV5ViyF16cJPT4taWNSaTLS64Dp67wuXigPZ &
+IDXR_PID=$(pgrep mina-indexer)
+sleep 3
+shutdown
 
 # Indexer server starts up on a minimal selection of blocks
-dl_mainnet 50 $TMP/blocks
+dl_mainnet 50 $BLOCKS_DIR
 server cli \
-    --startup-dir $TMP/blocks \
-    --watch-dir $TMP/blocks \
+    --startup-dir $BLOCKS_DIR \
+    --watch-dir $BLOCKS_DIR \
     --initial-ledger tests/ledgers/mainnet-genesis.json \
     --is-genesis-ledger \
     --root-hash 3NKeMoncuHab5ScarV5ViyF16cJPT4taWNSaTLS64Dp67wuXigPZ &
-IDXR_PID=$(echo $!)
+IDXR_PID=$(pgrep mina-indexer)
```

Comment:
regression?

Reply:
`echo $!` was grabbing the wrong PID because of the subshell. So I reverted my suggestion

Reply:
I'm seeing the same thing